pom.xml

```diff
@@ -63,7 +63,7 @@
                                     <arg line="-DCMAKE_TOOLCHAIN_FILE=${basedir}/toolchains/linux-amd64.cmake -DCMAKE_BUILD_TYPE=Release ${basedir}/src/main/c"/>
                                 </exec>
                                 <exec executable="cmake" dir="${project.build.directory}/native/linux/amd64" failonerror="true">
-                                    <arg line="--build . --config Release"/>
+                                    <arg line="--build . --config Release -j 16"/>
```

Comment:
I am pretty sure cmake just detects this + I am not sure how much faster this really is (I can only compile the linux one, but it took around 5 seconds)